crates/manifest/src/lib.rs

```diff
@@ -158,6 +158,8 @@ pub struct WasmConfig {
     pub mounts: Vec<DirectoryMount>,
     /// Optional list of HTTP hosts the component is allowed to connect.
     pub allowed_http_hosts: Vec<String>,
+    /// Flag to enable access to the Spin home directory
+    pub enable_monitoring: bool,
```

Comment:
The name seems out of sync with the documentation...?  (I don't fully understand the implementation though so I'm probably missing something!)

Reply:
No you're not missing anything, I've been too lazy and haven't updated the SIP

Reply:
All updated now :) It should be nice and clear.

Reply:
@ecumene just pinging on this, I'm still seeing a mismatch between the flag name and flag doc string...?

Reply:
I didn't see your reply:

I bounced the name `monitoring` back and forth with @lann. I suggested it but I don't think it's an accurate description either. I kept it hoping someone would come up with something better.

The flag name should be changed, I agree. How about something like `enable_home` or `mount_home`? I think someone with more experience than I could chime in

Reply:
`mount_home_dir` or `access_home_dir` perhaps?  I'm going by the doc string, though, I don't really understand the feature, so I could be way off beam!  From the screenshot... this is allowing the hosted modules access to each others' asset directories?  In which case `access_other_component_dirs` might be clearer than "home"...?

Reply:
(Many apologies if I am being obtuse here... I haven't followed enough and should have avoided suggesting things from a position of ignorance!)

Reply:
The current iteration of this feature has a very straightforward effect: the `~/.spin/<your-app>/` dir is mounted in the component's WASI env as `/.spin/`. Currently the only useful files we expect there are the `logs/<component>_std{err,out}.txt`s, but anything that Spin wants to put in that dir would be available, which is why I'm fine with the more general "monitoring" term.